x-pack/plugins/ingest_manager/server/services/epm/packages/install.ts

```diff
@@ -42,35 +42,42 @@ export async function installLatestPackage(options: {
 export async function ensureInstalledDefaultPackages(
   savedObjectsClient: SavedObjectsClientContract,
   callCluster: CallESAsCurrentUser
-): Promise<void> {
+): Promise<Installation[]> {
+  const installations = [] as Installation[];
```

Comment:
```suggestion
  const installations: Installation[] = []
```
any particular reason for using a type assertion here?